### ISSUE TITLE
Issue in POST /articles api endpoint: tags are not returned in the initial creation response

### ISSUE THREAD

**Issue by nishantwrp** [forem/forem#11516]
**Describe the bug**

I'm using API to create an article. I'm sending the tags as a list of strings in the request payload. But, can't find the tags in the response or in the articles on dev.to. 

**To Reproduce**

1. Send an API request to /articles for creating a article. 
For eg.
```
.{
  title: 'Quickwork Testing',
  body_markdown: '# asdasd',
  published: false,
  series: undefined,
  main_image: 'https://avatars3.githubusercontent.com/u/36989112?s=460&v=4',
  canonical_url: 'https://dasdasda.com',
  description: 'dsafadfsd',
  tags: [ 'javascript' ],
  organization_id: NaN
}

```

2. This the response received.

```
{
  "type_of": "article",
  "id": 520013,
  "title": "Quickwork Testing",
  "description": "dsafadfsd",
  "readable_publish_date": "Nov 20",
  "slug": "quickwork-testing-ape",
  "path": "/nishantwrp/quickwork-testing-ape",
  "url": "https://dev.to/nishantwrp/quickwork-testing-ape",
  "comments_count": 0,
  "public_reactions_count": 0,
  "published_timestamp": "2020-11-20T11:05:48Z",
  "positive_reactions_count": 0,
  "cover_image": "https://res.cloudinary.com/practicaldev/image/fetch/s--q96skcRN--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://avatars3.githubusercontent.com/u/36989112%3Fs%3D460%26v%3D4",                                                                                                                                                                      
  "social_image": "https://res.cloudinary.com/practicaldev/image/fetch/s--184jV2TS--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://avatars3.githubusercontent.com/u/36989112%3Fs%3D460%26v%3D4",                                                                                                                                                                     
  "canonical_url": "https://dasdasda.com",
  "created_at": "2020-11-20T11:05:48Z",
  "published_at": "2020-11-20T11:05:48Z",
  "last_comment_at": "2020-11-20T11:05:48Z",
  "tag_list": "",
  "tags": [],
  "body_html": "<h1>\n  <a name=\"asdasd\" href=\"#asdasd\" class=\"anchor\">\n  </a>\n  asdasd\n</h1>\n\n",
  "body_markdown": "# asdasd",
  "user": {
    "name": "Nishant Mittal",
    "username": "nishantwrp",
    "twitter_username": "nishantwrp",
    "github_username": "nishantwrp",
    "website_url": "https://www.nishantwrp.com",
    "profile_image": "https://res.cloudinary.com/practicaldev/image/fetch/s--UUPQ1U-G--/c_fill,f_auto,fl_progressive,h_640,q_auto,w_640/https://dev-to-uploads.s3.amazonaws.com/uploads/user/profile_image/239671/98b595a2-72f3-4814-9e6d-b14ede18cfff.jpeg",
    "profile_image_90": "https://res.cloudinary.com/practicaldev/image/fetch/s--ffdeoH58--/c_fill,f_auto,fl_progressive,h_90,q_auto,w_90/https://dev-to-uploads.s3.amazonaws.com/uploads/user/profile_image/239671/98b595a2-72f3-4814-9e6d-b14ede18cfff.jpeg"
  }
}

```

**Expected behavior**

It can be seen that the tags property is empty array but shouldn't be.

**Screenshots**

**Desktop (please complete the following information):**

- OS, version:
- Browser, version:

**Smartphone (please complete the following information):**

- Device:
- OS, version:
- Browser, version:

**Additional context**

**Comment by rhymes** [comment]
@nishantwrp this sounds like a legit bug. Can you verify if the tags have actually been added to the article? What if you fetch the article again, do they appear there?

**Comment by nishantwrp** [comment]
I've done that. They do not.

**Comment by rhymes** [comment]
@nishantwrp ok so the issue isn't serialization but it's upon creation, that's a clue. Thanks for the report!

**Comment by nishantwrp** [comment]
Hi, closing this issue. Because this was an error from my side. The request body should've been
```
.article: {
    title: 'Quickwork Testing',
    body_markdown: '# asdasd',
    published: false,
    series: undefined,
    main_image: 'https://avatars3.githubusercontent.com/u/36989112?s=460&v=4',
    canonical_url: 'https://dasdasda.com',
    description: 'dsafadfsd',
    tags: [ 'javascript' ],
    organization_id: NaN
  }
```

but I was sending
```
{
  title: 'Quickwork Testing',
  body_markdown: '# asdasd',
  published: false,
  series: undefined,
  main_image: 'https://avatars3.githubusercontent.com/u/36989112?s=460&v=4',
  canonical_url: 'https://dasdasda.com',
  description: 'dsafadfsd',
  tags: [ 'javascript' ],
  organization_id: NaN
}
```

But why wasn't there any error? I tested this thing locally and there was an error that I didn't send an article object.
/cc @rhymes

**Comment by nishantwrp** [comment]
Closes #11692
